### DISCUSSION TITLE
Option to choose biome in single missions.

### DISCUSSION THREAD

**Discussion by MarkCringan** [FakeFishGames/Barotrauma#13439] [Ideas and suggestions]
I think it would be nice to have an option to choose what biome(s) we want to play for doing singular missions in multiplayer. 

If somebody does not want to play on the Europan Ridge, they should be able to opt out of it.

If somebody only wants to play games in the Great Sea, that should be an option too!

**Comment by jigardi** [comment]
Makes a lot of sense I think,  especially when you consider that most people don't get too far in the campaign.
